playground/pages/index.vue

```diff
@@ -33,6 +33,7 @@ export default {
       images: [
         { src: 'cloudinary:/remote/nuxt-org/blog/going-full-static/main.png', alt: 'Cloudinary' },
         { src: 'fastly:/image.jpg', alt: 'fastify' },
+        { src: 'imgix:/examples/bluehat.jpg', alt: 'imgix'},
```

Comment:
Could you fix lint error?

Reply:
Ya, sorry about that. Fixed the lint error.